packages/react/package.json

```
@@ -28,7 +28,10 @@
     "eslint:fix": "eslint src --ext js,jsx,ts,tsx --fix",
     "prettier": "prettier \"src/**/*.!(js|jsx|ts|tsx)\" --check",
     "prettier:fix": "prettier \"src/**/*.!(js|jsx|ts|tsx)\" --write",
-    "tsc": "tsc --noEmit"
+    "tsc": "tsc --noEmit",
+    "test": "vitest",
+    "test:ui": "vitest --ui",
+    "coverage": "vitest run --coverage"
```

Préférence perso
```suggestion
    "test:ci": "vitest run --coverage"
```
car en fin de compte on ce va l'utiliser que dans la ci principalement